Resources/Prototypes/_StarLight/Entities/Objects/Weapons/Guns/LMGs/lmgs.yml

```
@@ -0,0 +1,42 @@
+        ejectSound: /Audio/Weapons/Guns/MagOut/batrifle_magout.ogg
+        priority: 2
+        whitelist:
+          tags:
+            - LightRifleHeavy
+      gun_chamber:
+        name: Chamber
+        startingItem: CartridgeLightRifle
+        priority: 1
+        whitelist:
+          tags:
+            - CartridgeLightRifle
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)